packages/app/src/runner/index.ts

```diff
@@ -186,6 +187,29 @@ export async function teardown () {
   isTorndown = true
 }
 
+/**
+ * Add a cross origin iframe for cy.origin support
+ */
+export function addCrossOriginIframe (location) {
+  const id = `Spec Bridge: ${location.originPolicy}`
+
+  // if it already exists, don't add another one
+  if (document.getElementById(id)) {
+    getEventManager().notifyCrossOriginBridgeReady(location.originPolicy)
+
+    return
+  }
+
+  addIframe({
+    id,
+    // the cross origin iframe is added to the document body instead of the
+    // container since it needs to match the size of the top window for screenshots
+    $container: document.body,
```

Comment:
`Cypress.$` is not instantiated sometimes when this function is called, so the body is just `document.body` over `$(document.body) which can be seen in the [original](https://github.com/cypress-io/cypress/pull/18075/files#diff-bd8f567e7cefe6212c99b7760b613f12ffa518138f1d19aff5f3e5cf0c677bf9R189)